### ISSUE TITLE
Can i get list of viewers pressed button on story?

### ISSUE THREAD

**Issue by deniben** [instagram4j/instagram4j#549]
Hi, can i do it, any ideas?

**Comment by jvogit** [comment]
What button, like a poll?

I believe you'd have to parse out the json for that info.

**Comment by deniben** [comment]
> What button, like a poll?
> 
> I believe you'd have to parse out the json for that info.

yes poll, like 'yes, no' and get it list of users and results how many yes or no
